model/swagger.yml

```diff
@@ -408,17 +231,14 @@ paths:
         '501':
           $ref: "#/components/responses/Error"
     parameters:
-      - name: identifier
-        in: path
-        description: lidvid or lid
-        required: true
-        schema:
-          type: string
+      - $ref: "#/components/parameters/Fields"
+      - $ref: "#/components/parameters/Identified"
   /collections/{identifier}/all:
     get:
       tags:
         - collections
-      summary: Get one collection by lidvid, or all versions of a collection by lid
+      summary: |
+        returns all collection versions (lid) with the identifier selected from all PDS collections. See identifier for details on how it determines what is returned.
```

Comment:
Should that be vid here instead of lid ? vid is for version identifier. 

lid is for logical identifier, maybe that (lid) should be after identifier instead ? Like "returns all collection versions  with the identifier (lid) selected from a ..."

I am not sure.

Reply:
No, lid but it should be after identifier. Fixed.